core/src/ast_builder/mod.rs

```diff
@@ -0,0 +1,58 @@
+mod delete;
+mod expr;
+mod expr_list;
+mod select;
+mod select_item_list;
+
+pub use {
+    delete::DeleteNode,
+    expr::{
+        aggregate::{max, sum, AggregateNode},
+        function::{abs, FunctionNode},
+    },
+    expr::{col, expr, nested, num, text, ExprNode},
+    expr_list::ExprList,
+    select::{
+        GroupByNode, HavingNode, LimitNode, LimitOffsetNode, OffsetLimitNode, OffsetNode,
+        ProjectNode, SelectNode,
+    },
+    select_item_list::SelectItemList,
+};
+
+pub fn table(table_name: &str) -> TableNode {
```

Comment:
[Question]
Is there a reason for `Table` residing in the mod.rs?  I expected separated file for example `table.rs` when I reading the code.

Reply:
Thanks again 👍 this is also really a good point that I missed.
I splitted the codes and add some comments, too.
![image](https://user-images.githubusercontent.com/2025065/177581116-22911278-d4bc-490f-84b9-0f45103f7837.png)